vertical-pod-autoscaler/pkg/recommender/model/cluster.go

```diff
@@ -348,15 +349,20 @@ func (cluster *ClusterState) findOrCreateAggregateContainerState(containerID Con
 	return aggregateContainerState
 }
 
-func (cluster *ClusterState) garbageCollectAggregateCollectionStates(now time.Time) {
+// GarbageCollectAggregateCollectionStates removes obsolete AggregateCollectionStates from the ClusterState.
+// AggregateCollectionState is obsolete in following situations:
+// 1) It has no samples and there are no more contributive pods,
```

Comment:
Please add a short explanation of what are "contributive pods"

Reply:
Added.